src/Command/Analyze.php

```diff
@@ -253,4 +254,21 @@ private function loadStrategy($type, array $options = null)
 
         return $strategy;
     }
+
+    /**
+     * Lookup for config file in current working directory.
+     *
+     * @return string
+     */
+    private function getDefaultConfigFilePathl()
```

Comment:
I guess the **L** after Path is a typo?

Reply:
Yes, you are right, thanks. I've fixed it.